src/codemodder/registry.py

```diff
@@ -59,14 +59,24 @@ def match_codemods(
         self,
         codemod_include: Optional[list] = None,
         codemod_exclude: Optional[list] = None,
+        sast_only=False,
     ) -> list[BaseCodemod]:
         codemod_include = codemod_include or []
         codemod_exclude = codemod_exclude or DEFAULT_EXCLUDED_CODEMODS
 
+        if sast_only:
```

Comment:
Feels like you could move the conditional to be within the list comprehension to avoid the duplication.